### ISSUE TITLE
map markers do not show up on mini map or full size map

### ISSUE THREAD

**Issue by bluenote73** [blish-hud/Blish-HUD#364]
Sorry if this is not a bug but an enhancement request! I get 0 markers on the mini map, or full size map, and I'm trying with several different packages.

Blish is really useful, and fast, would really love if this was fixed or added! 
thanks!

**Comment by sho-87** [comment]
I would love to see this as well

**Comment by dlamkins** [comment]
Yes, this is a feature TacO added that we have not yet added.  We have begun a major rewrite of the markers & paths module just recently and this feature is on the roadmap for this new version of the module.  So far we have markers rendering successfully and hope to have trails rendering just the same in the near future.

I'll update here once that module hits release.  In the meantime, you can always try this new version of the module by joining the Discord (see the Readme) and checking in the #module_discussion channel.

**Comment by dlamkins** [comment]
Please note that the above mentioned version of the module is now generally available in our #prerelease_builds Discord channel with hopes to release a full version planned for v0.8.0 of Blish HUD.
